include/bluetooth/mesh/light_xyl_cli.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_light_xyl_cli Light xyL Client model
+ * @{
+ * @brief API for the Light xyL Client model.
+ */
+
+#ifndef BT_MESH_LIGHT_XYL_CLI_H__
+#define BT_MESH_LIGHT_XYL_CLI_H__
+
+#include <bluetooth/mesh/light_xyl.h>
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct bt_mesh_light_xyl_cli;
+
+/** @def BT_MESH_MODEL_LIGHT_XYL_CLI
+ *
+ * @brief Light XYL Client model composition data entry.
+ *
+ * @param[in] _cli Pointer to a @ref bt_mesh_light_xyl_cli instance.
+ */
+#define BT_MESH_MODEL_LIGHT_XYL_CLI(_cli)                                      \
+	BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_LIGHT_XYL_CLI,                       \
+			 _bt_mesh_light_xyl_cli_op, &(_cli)->pub,              \
+			 BT_MESH_MODEL_USER_DATA(struct bt_mesh_light_xyl_cli, \
+						 _cli),                        \
+			 &_bt_mesh_light_xyl_cli_cb)
+
+/** Light xyL Client state access handlers. */
+struct bt_mesh_light_xyl_cli_handlers {
```

Comment:
this is not solved. However, this is a minor style comment. Up to you.

Reply:
Ok, then I think I misunderstood the comment. If you care to elaborate I can give it another go, but since you say it is minor we could also just skip it.